# Robot states

bool is_safe
uint8 controller_mode
uint8 gait_type
time timestamp
